### PR TITLE
[chart/solr] Description: Updated Version and Readme

### PR DESCRIPTION
**Pull request by roachmd**
Bumped Version 1.1.1
	Bumped AppVersion 7.7.2
	Updated Readme with examples

 On branch master
 Your branch is ahead of 'origin/master' by 22 commits.
   (use "git push" to publish your local commits)

 Changes to be committed:
	modified:   Chart.yaml
	modified:   README.md
	modified:   values.yaml

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: incubator/solr/Chart.yaml [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 apiVersion: "v1"
 name: "solr"
-version: "1.1.0"
-appVersion: "7.6.0"
+version: "1.1.1"
```
Comment (maorfr) [inline incubator/solr/Chart.yaml]: please bump the minor version

File: incubator/solr/values.yaml [hunk 2]
```diff
@@ -22,12 +22,12 @@ terminationGracePeriodSeconds: 180
...
 image:
   repository: solr
-  tag: 7.6.0
+  tag: 7.7.2
   pullPolicy: IfNotPresent
 
 # Solr pod liveness
 livenessProbe:
-  initialDelaySeconds: 20
+  initialDelaySeconds: 45
```
Comment (maorfr) [inline incubator/solr/values.yaml:30]: why is this required?

Comment (w2g-roachmd) [reply incubator/solr/values.yaml:30]: Running this on my 16GB mac in minikube the start times were longer.
